Makefile

```diff
@@ -1,16 +1,29 @@
 # Extern dependencies commit hashes
 filecoin-ffi-commit = 5d00bb4365a97890
```

Comment:
while you're at this, can we change this from `filecoin-ffi-commit` to `filecoin-ffi-branch` and update the comment above it? it turns out filecoin-ffi just names their release branches by the commit hash, i g confused about that when i first wrote this.

Reply:
also, we probably should change it to snake_case for consistency^^

Reply:
done, but I had to add a second `rev-parse` to the check to properly handle branches and not just commits (the current practice in filecoin-ffi of branchname==commit hash is convenient because it means we can pretend it's a commit hash) and remove the `findstring` to do proper equality; basically the same logic though